langchain/package.json

```diff
@@ -1378,6 +1381,7 @@
   },
   "dependencies": {
     "@anthropic-ai/sdk": "^0.9.1",
+    "@google-cloud/local-auth": "^3.0.0",
```

Comment:
Add as peer + devDependency:

https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md

Reply:
Why is this using @google-cloud/local-auth instead of google-auth-library?
(Why does Google have so many different ways to do the same exact thing?)